boards/realtek/rtl872xda_evb/rtl872xda_evb_defconfig

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2024 Realtek Semiconductor Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_RUNTIME_NMI=y
+
+CONFIG_CORTEX_M_SYSTICK=y
+
+# enable rcc
+CONFIG_CLOCK_CONTROL=y
+CONFIG_PINCTRL=y
```

Comment:
Kconfigs select this

Reply:
Not addressed

Reply:
Fixed, I misunderstood and just changed line 8.